### ISSUE TITLE
There are sudden change in validation line

### ISSUE THREAD

**Issue by xiaoerlaigeid** [tensorflow/tensorflow#12639]
------------------------

### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
- **Os platform version**: Win10
- **install From conda:
- **tensorflow version** :tensorflow 1.1
- **Python version**: 3.5
- **Bazel version (if compiling from source)**:
- **CUDA/cuDNN version**: 8.0 5.0
- **GPU model and memory**: GTX1080ti

### Describe the problem
[I resume weights model from local. I find there are sudden change in validation line, but my train line don't this problem. I think maybe there are something wrong with tensorflow source code.]
below is train history graph
(https://github.com/xiaoerlaigeid/face-and-Pedestrian-detection-/blob/master/Figure_1.png)

### Source code / logs
`		if RESUME:
			print('Restoring previously trained model at %s' % MODEL_SAVE_PATH)
			saver.restore(sess, MODEL_SAVE_PATH)

			# Restore previous loss history
			with open('loss_history.p', 'rb') as f:
				loss_history = pickle.load(f)
		else:
			print('Training model from scratch')
			# Variable initialization
			sess.run(tf.global_variables_initializer())

			# For book-keeping, keep track of training and validation loss over epochs, like such:
			# [(train_acc_epoch1, valid_acc_epoch1), (train_acc_epoch2, valid_acc_epoch2), ...]
			loss_history = []`

**Comment by jart** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
